alienvault/resource_sensor.go

```diff
@@ -106,15 +106,15 @@ func resourceSensorDelete(d *schema.ResourceData, m interface{}) error {
 }
 
 func flattenSensor(sensor *alienvault.Sensor, d *schema.ResourceData) {
-	d.SetId(sensor.UUID)
+	d.SetId(sensor.V1ID)
 	d.Set("name", sensor.Name)
 	d.Set("description", sensor.Description)
 	d.Set("activation_code", sensor.ActivationCode)
 }
 
 func expandSensor(d *schema.ResourceData) *alienvault.Sensor {
 	sensor := &alienvault.Sensor{}
-	sensor.UUID = d.Id()
+	sensor.V1ID = d.Id()
```

Comment:
Will have to set both ID versions to support v1 and v2 which is a bit gross but necessary:
```suggestion
	sensor.V1ID = d.Id()
	sensor.V2ID = d.Id()
```